packages/gestalt-design-tokens/tokens/vr-theme-web-mapping/comp-web-color-light.json

```diff
@@ -124,6 +124,34 @@
             "value": "{sema.color.background.disabled.value}"
           }
         },
+        "dark": {
+          "default": {
+            "value": "{sema.color.background.dark.value}"
+          },
+          "hover": {
+            "value": "{sema.color.hover.background.dark.value}"
+          },
+          "active": {
+            "value": "{sema.color.pressed.background.dark.value}"
+          },
+          "disabled": {
```

Comment:
i dont think we need the disabled one. 

I think disabled is common to all the colors. 

![Screenshot by Dropbox Capture](https://github.com/user-attachments/assets/f0a96e61-cb34-4fb0-8300-42fb0a9645c0)